src/drivers/stm32h.h

```diff
@@ -12,26 +12,29 @@ struct mg_tcpip_driver_stm32h_data {
   //    100-150 MHz   HCLK/62        1
   //    20-35 MHz     HCLK/16        2
   //    35-60 MHz     HCLK/26        3
-  //    150-250 MHz   HCLK/102       4  <-- value for Nucleo-H* on max speed driven by HSI
-  //    250-300 MHz   HCLK/124       5  <-- value for Nucleo-H* on max speed driven by CSI
+  //    150-250 MHz   HCLK/102       4  <-- value for max speed HSI
+  //    250-300 MHz   HCLK/124       5  <-- value for Nucleo-H* on CSI
```

Comment:
First one was redundant, but second one now is imprecise, the processor con run from CSI but not necessarily at max speed...

Reply:
What would be the correct comment ?

Reply:
Maybe "max speed on HSI" "max speed on CSI" is concise and clear enough ? WDYT ?

Reply:
You've got the right comment in the other PR!